storage/aws/elasticache/variables.tf

```
@@ -93,6 +93,12 @@ variable "log_retention_in_days" {
   default     = 0
 }
 
+variable "max_memory_samples" {
+  description = "Number of samples to check for every eviction"
+  type        = string
+  default     = ""
```

String is a weird number type...
```suggestion
  type        = number
  default     = null
```